packages/app/src/hooks/useUnstaking/index.tsx

```diff
@@ -25,15 +30,27 @@ export const useUnstaking = () => {
   const { ss58 } = getStakingChainData(network)
   const { nominees } = getNominationStatus(activeAddress, 'nominator')
 
+  // Calculate if this is a legacy nominator
```

Comment:
What is the definition of a legacy nominator - one that is staking with less than the minimum to earn rewards?